exercises/count/exercise.js

```diff
@@ -15,7 +15,7 @@ var db, url = 'mongodb://localhost:27017/learnyoumongo'
 exercise.addSetup(function(mode, cb) {
   var self = this
   var numbers = [1, 3, 10]
-  var number = numbers[Math.floor(Math.random() * numbers.length)]
+  var number = numbers[Math.floor(Math.random() * (numbers.length-1))]
```

Comment:
The problem with checking for ages > 10 is that it will return 0.

0 is also returned by invalid queries, so if the user is doing something horribly wrong (like not removing `process.argv[2]` from the url from the previous exercise) that error will be masked and the exercise will pass.